immuneML/reports/ml_reports/GeneratorReport.py

```diff
@@ -0,0 +1,86 @@
+import plotly.graph_objects as go
+
+from pathlib import Path
+from numpy import array
+from immuneML.data_model.dataset.Dataset import Dataset
+from immuneML.ml_methods.UnsupervisedMLMethod import UnsupervisedMLMethod
+from immuneML.reports.ReportOutput import ReportOutput
+from immuneML.reports.ReportResult import ReportResult
+from immuneML.reports.ml_reports.UnsupervisedMLReport import UnsupervisedMLReport
+from immuneML.util.PathBuilder import PathBuilder
+import numpy as np
+import pandas as pd
+import matplotlib.pylab as plt
+import logomaker
+plt.style.use("seaborn")
+
+import plotly.graph_objs as go
+
+
+class GeneratorReport(UnsupervisedMLReport):
+    @classmethod
+    def build_object(cls, **kwargs):
+        name = kwargs["name"] if "name" in kwargs else "GeneratorReport"
+        return GeneratorReport(name=name)
+
+    def __init__(self, dataset: Dataset = None, method: UnsupervisedMLMethod = None, result_path: Path = None,
+                 name: str = None, number_of_processes: int = 1):
+        super().__init__(dataset=dataset, method=method, result_path=result_path,
+                         name=name, number_of_processes=number_of_processes)
+        self.dataset = dataset
+
+    def _generate(self) -> ReportResult:
+
+        generated_sequences = self.result_path / f"{self.name}GeneratedSequences.csv"
+
+        data = pd.DataFrame(self.method.generated_sequences, columns=["Generated Sequences"])
```

Comment:
why store generated sequences in the method instead of generating the sequences here by calling method.generate()? :)